hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoCreateHandler.java

```diff
@@ -260,10 +260,10 @@ private Account buildAccount(CryptoCreateTransactionBody op, HandleContext handl
             builder.key(op.keyOrThrow()).alias(op.alias());
         }
 
-        if (op.hasStakedAccountId() || op.hasStakedNodeId()) {
-            final var stakeNumber =
-                    getStakedId(op.stakedId().kind().toString(), op.stakedNodeId(), op.stakedAccountId());
-            builder.stakedNumber(stakeNumber);
+        if (op.hasStakedAccountId()) {
```

Comment:
Why do we have a different logic here than below?

Same question also do we have any option to have both?

Reply:
It's the same logic beside creating doesn't try to remove stakedAccountId. You can only have one of them.